source/grid/grid_tools.cc

```diff
@@ -167,6 +167,53 @@ namespace GridTools
   }
 
 
+
+  template <>
+  double
+  cell_measure<1, 1>
+  (const std::vector<Point<1> > &all_vertices,
+   const unsigned int (&vertex_indices)[GeometryInfo<1>::vertices_per_cell])
+  {
+    return all_vertices[vertex_indices[1]][0]
+           - all_vertices[vertex_indices[0]][0];
+  }
+
+
+
+  template <>
+  double
+  cell_measure<1, 2>
+  (const std::vector<Point<2> > &,
+   const unsigned int (&)[GeometryInfo<1>::vertices_per_cell])
+  {
+    Assert(false, ExcNotImplemented());
+    return std::numeric_limits<double>::quiet_NaN();
+  }
+
+
+
+  template <>
+  double cell_measure<1, 3>
+  (const std::vector<Point<3> > &,
+   const unsigned int (&)[GeometryInfo<1>::vertices_per_cell])
+  {
+    Assert(false, ExcNotImplemented());
+    return std::numeric_limits<double>::quiet_NaN();
+  }
+
+
+
+  template <>
+  double cell_measure<2, 3>
+  (const std::vector<Point<3> > &,
+   const unsigned int (&)[GeometryInfo<2>::vertices_per_cell])
+  {
+    Assert(false, ExcNotImplemented());
+    return std::numeric_limits<double>::quiet_NaN();
+  }
+
+
+
   template <>
   double
   cell_measure<3>(const std::vector<Point<3> > &all_vertices,
```

Comment:
That function was already there. I will make it consistent with the others (that change does make the code prettier).